src/lib/createPollBlocks.ts

```diff
@@ -31,6 +31,15 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
 
     block.addDividerBlock();
 
+    // Array of vote quantities
+    const voteQuantities = poll.votes.map(vote => vote.quantity)
+    
+    // Forms array of option indices with maximum votes (more than 1 option can be max-voted)
+    const maxVoteIndices = voteQuantities.reduce(function(ind: number[], el, i) { 
+        if (el === Math.max(...voteQuantities)) 
```

Comment:
That's very inefficient. You're calling the Max function for all element in array. Could've saved that info in solve variable

Reply:
Resolved. Thanks.